### PR TITLE
[matex] add model longformer

### PR DESCRIPTION
**Pull request by happyxuwork**

### CODE REVIEW

**Comment by shh2000** [comment]
please change target to FlagOpen:longformer-fixpatch

File: training/metax/longformer-pytorch/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| 训练吞吐量     | p_train,见“性能指标”    | 不包含每个epoch末尾的评估部分耗时           |
+| **计算吞吐量** | **p_core,见“性能指标”** | 不包含数据IO部分的耗时(p3>p2>p1)            |
+| 训练结果       | mlm_acc,见“性能指标”    | masked_lm任务准确率                         |
+| 额外修改项     | 无                      |                                             |
+
+* 性能指标
+
+| 配置              | precision | fix_hp | e2e_time | p_whole | p_train | p_core | mlm_acc | mem     |
+|-----------------|-----------|--------|----------|---------|---------|--------|---------|---------|
+| A100单机单卡（1x1） | fp32      | bs=16  |         |       |       |      |        | 24.1/64.0 |
```
Comment (shh2000) [inline training/metax/longformer-pytorch/README.md:46]: please add "/" to vacant cells, represent "secret" rather than "not implemented"

Comment (happyxuwork) [reply training/metax/longformer-pytorch/README.md]: ok  already alter

**Comment by happyxuwork** [comment]
> please change target to FlagOpen:longformer-fixpatch

ok

**Pull request closed without merging**